pyaedt/generic/compliance.py

```diff
@@ -28,6 +28,16 @@
 
 
 class VirtualCompliance:
+    """Virtual compliance class.
+
+    Parameters
+    ----------
+    desktop : :class:``pyaedt.desktop.Desktop``
+        Desktop object
+    template : str
+        Full path to the template. Supported formats are json and toml.
```

Comment:
```suggestion
        Desktop object.
    template : str
        Full path to the template. Supported formats are JSON and TOML.
```
File extensions are written in all capital letters.